appinventor/appengine/src/com/google/appinventor/client/editor/simple/palette/ComponentHelpWidget.java

```diff
@@ -7,6 +7,9 @@
 package com.google.appinventor.client.editor.simple.palette;
 
 import com.google.appinventor.client.ComponentsTranslation;
+import com.google.gwt.user.client.Command;
```

Comment:
Make sure that the imports are ordered appropriately. The gwt import here should be further down with the other gwt imports